HideButtonGlow.lua

```diff
@@ -99,9 +99,11 @@ if ElvUI then
 	if LibCustomGlow and LibCustomGlow.ShowOverlayGlow then
 		local OriginalShowOverlayGlow = LibCustomGlow.ShowOverlayGlow
 		function LibCustomGlow.ShowOverlayGlow(self)
-			local spellId = self:GetSpellId()
-			if spellId and addon:ShouldHideGlow(spellId) then
-				return
+			if self:GetSpellId() then
```

Comment:
If the `GetSpellId` function doesn't exist then it seems like would just error here instead.

I think you can remove the `if self:GetSpellId() then` wrapper you added but change `local spellId = self:GetSpellId()` to `local spellId = self.GetSpellId and self:GetSpellId()`. That way if the function doesn't exist on whatever ElvUI is passing into here then it will just show the glow instead of erroring.